### ISSUE TITLE
sys:1: ResourceWarning: unclosed 

### ISSUE THREAD

**Issue by nyxynyx** [ranaroussi/fix-yahoo-finance#41]
Whenever I import the module

    import fix_yahoo_finance as yf

and without using any of the functions in `fix_yahoo_finance`, create a SQLAlchemy engine and use Pandas DataFrame's `to_sql`, 

it gives the error

    sys:1: ResourceWarning: unclosed <socket.socket fd=456, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.112', 49217), raddr=('192.168.1.115', 3306)>

Is this a bug?

	import pandas as pd
	from sqlalchemy import create_engine
	import fix_yahoo_finance as yf

	engine = create_engine("mysql+mysqlconnector://user:password@server")
	df = pd.DataFrame()
	df.to_sql(
		con=engine,
		name='yahoo',
		if_exists='append',
		index=False)

Use this code to reproduce the error/warning. Removing `fix_yahoo_finance` will remove the error.

**Comment by nyxynyx** [comment]
import pandas as pd
	from sqlalchemy import create_engine
	import fix_yahoo_finance as yf

	engine = create_engine("mysql+mysqlconnector://user:password@server")
	df = pd.DataFrame()
	df.to_sql(
		con=engine,
		name='stocksYahoo',
		if_exists='append',
		index=False,
		chunksize=1000)

Use this code to reproduce the error/warning. Removing `fix_yahoo_finance` will remove the error.

**Comment by jainraje** [comment]
I just updated from using a older version (0.0.8). I'm now seeing this same warning. Not sure what this warning implies and why it is there now?

..../anaconda/lib/python3.5/site-packages/pandas_datareader/data.py:137: ResourceWarning: unclosed <socket.socket fd=62, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.6', 49817), raddr=('172.217.11.174', 80)>
  session=session).read()

**Comment by nyxynyx** [comment]
Great so its not just me!

These are the versions of some of the modules that I am using

```
fix-yahoo-finance==0.0.19
pandas==0.20.3
pandas-datareader==0.5.0
requests==2.18.4
```
